cmake.deps/cmake/BuildWasmtime.cmake

```diff
@@ -3,6 +3,9 @@ get_externalproject_options(wasmtime ${DEPS_IGNORE_SHA})
 ExternalProject_Add(wasmtime
   DOWNLOAD_DIR ${DEPS_DOWNLOAD_DIR}/wasmtime
   SOURCE_SUBDIR crates/c-api
-  CMAKE_ARGS ${DEPS_CMAKE_ARGS} -D WASMTIME_FASTEST_RUNTIME=ON
+  CMAKE_ARGS ${DEPS_CMAKE_ARGS}
+    -D WASMTIME_FASTEST_RUNTIME=ON
+    -D WASMTIME_DISABLE_ALL_FEATURES=ON
+    -D WASMTIME_FEATURE_CRANELIFT=ON
```

Comment:
Can we document why we want these?

Reply:
To make wasmtime go *brrrrr*?

Reply:
* `FASTEST_RUNTIME` is basically `Release` on steroids (full lto)
* `DISABLE_ALL_FEATURES` does what it says on the tin -- a lot of optional stuff we don't need, except...
* `CRANELIFT`, which is the low-level code generator (required to compile wasm to platform code; otherwise no worky)